resources/helm/overlays/istio_cni/templates/configmap.yaml

```diff
@@ -47,4 +47,17 @@ data:
           "exclude_namespaces": [ "{{ .Release.Namespace }}" ]
       }
     }
+  cni_network_config_v2_1: |-
+    {
+      "cniVersion": "0.3.0",
+      "name": "v2-1-istio-cni",
+      "type": "v2-1-istio-cni",
+      "log_level": "info",
+      "kubernetes": {
+          "kubeconfig": "__KUBECONFIG_FILEPATH__",
+          "cni_bin_dir": "/opt/multus/bin",
+          "netns_setup_executable": "v2-1-istio-iptables",
```

Comment:
This might be not needed (https://github.com/maistra/istio/pull/273 for context)

Reply:
It is needed as this is the only way to get configuration into the CNI plugin.